README.md

```diff
@@ -21,6 +21,7 @@ the "what" and "why" of data breach alerts.
 
 - [Volta](https://volta.sh/) (installs the correct version of Node and npm)
 - [Postgres](https://www.postgresql.org/) | Note: On a Mac, we recommend downloading the [Postgres.app](https://postgresapp.com/) instead.
+- [Python](https://www.python.org/downloads/) | [With Homebrew](https://docs.brew.sh/Homebrew-and-Python)
```

Comment:
Interesting. I have Python3 installed via homebrew on macOS, but I'm not sure which version is installed by default these days, or which version is included via Xcode utils.

Reply:
I think the base system on macOS might be 3 now.